styles.css

```diff
@@ -0,0 +1,79 @@
+div.container {
+  width: 60rem;
+  height: 640px;
+}
+
+.row {
+  display: flexboxgrid;
+  flex-direction: column;
+}
+
+.col {
+  flex: 1;
+  margin: 0.1em;
+  padding: 0.8em;
+}
+
+div.difference-header {
+  margin-top: 50px;
+  margin-bottom: 15px;
+}
+
+h1 {
+  font-size: 1.9em;
+}
+
+.difference-images {
```

Comment:
Let your three-column layout determine the width. You can add horizontal margin to the `section` to "squish" all three columns together a bit.